Makefile

```diff
@@ -0,0 +1,73 @@
+GO_MODULE := $(shell git config --get remote.origin.url | grep -o 'github\.com[:/][^.]*' | tr ':' '/')
+REPO_SLUG := $(shell echo ${GO_MODULE} | cut -d/ -f2-)
+CMD_NAME := $(shell basename ${GO_MODULE})
+DEFAULT_APP_PORT ?= 8080
+GIT_COMMIT := $(shell git rev-parse HEAD)
+
```

Comment:
none of this works currently since it assumes the root is buildable, and shouldn't be necessary for this repo in general.
```suggestion
```

Reply:
removed